generators/app/templates/src/module.cc

```diff
@@ -1,40 +1,56 @@
 #include "<%= moduleHeaderFileName %>"
 
-using namespace Napi;
```

Comment:
Curious, why remove this?

Reply:
Accidentally, because I wanted to verify I used Napi:: in every statement. I will re-add the line before squashing.